java/crossbuild/build-win.bat

```diff
@@ -0,0 +1,16 @@
+:: install git, java 8, maven, visual studio community 15 (2017)
+
+set MSBUILD=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe

```

Comment:
hardcoded path?

Reply:
I know :( I am not an expert on windows setup. To the best of my attempt, this script can be seen partially as 'how to' or MS VS bin needs to be put on the path after installation, not sure how to automate it.